editor/js/Loader.js

```diff
@@ -572,4 +582,34 @@ var Loader = function ( editor ) {
 
 	}
 
+	function isGltf1( contents ) {
+
+		var resultContent;
+
+		if ( typeof contents === 'string' ) {
+
+			resultContent = contents;
+
+		} else {
+
+			var magic = THREE.LoaderUtils.decodeText( new Uint8Array( contents, 0, 4 ) );
+
+			if ( magic === 'glTF' ) {
```

Comment:
probably want `magic !== 'glTF'` here.

Reply:
Does this need to be resolved?

Reply:
It does yeah; the first four bytes are `glTF` for both 1.0 and 2.0. The next four bytes are a version, which must be `1` in glTF 1.0. Will add a comment below.